web-server/index.js

```diff
@@ -40,6 +40,10 @@ app.get('/', (request, response, next) => {
   response.renderMarkdownFile(`/README.md`)
 })
 
+app.get('/*', (request, response, next) => {
+  response.renderFile(request.path)
```

Comment:
this works but it allows people to render files they should not be able to. So lets do this

https://github.com/LearnersGuild/curriculum/commit/6eddf18a812d7ba87f85e9671d748a8e7013d93b

please resubmit the code above and I'll merge it